### PR TITLE
Issue #6966: aligned javadoc/xdoc for FinalParameters

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6966 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/xdocs/config_misc.xml [hunk 1]
```diff
@@ -923,7 +923,7 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           </tr>
           <tr>
             <td>ignorePrimitiveTypes</td>
-            <td>ignore primitive types as parameters</td>
+            <td>ignore primitive types as parameters.</td>
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: Haven't we been making these descriptions start with an uppercase letter.

File: src/xdocs/config_misc.xml [hunk 2]
```diff
@@ -895,10 +895,10 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
     </section>
 
     <section name="FinalParameters">
+      <p>Since Checkstyle 3.0</p>
       <subsection name="Description" id="FinalParameters_Description">
-        <p>Since Checkstyle 3.0</p>
         <p>
-          Check that parameters for methods, constructors, and catch blocks are
+          Check that parameters for methods, constructors, and catch, foreach blocks are
```
Comment (rnveach) [inline src/xdocs/config_misc.xml]: `and catch, foreach` => `catch, and foreach`
Should this be for-each or just foreach?

File: src/xdocs/config_misc.xml (hunk 1 above)
Comment (pbludov) [reply src/xdocs/config_misc.xml]: done

File: src/xdocs/config_misc.xml (hunk 2 above)
Comment (pbludov) [reply src/xdocs/config_misc.xml]: done. For other checks we use `for-each`

**Pull request merged**